external-docs/docs/guides/command-line/inspect-command.md

```diff
@@ -1,3 +1,939 @@
+  | Publisher     | Publisher information web page url.                                 |
+  | Landing Pages | List of web urls of landing pages.                                  |
+  | Themes        | List of web urls of theme information.                              |
+  | Keywords      | List of keywords.                                                   |
+  | Contact Point | Contact information.                                                |
+  | Identifier    | Data set identifier.                                                |
+  | Comment       | Any comments associated with the data set.                          |
+  | Description   | Formatted description of the data set.                              |
+
+* `The data cube has the following data structure definition`:
+
+  | Field                                 | Description                                                                                                   |
+  | ------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+  | Dataset Label                         | Label of the dataset.                                                                                         |
+  | Number of Components                  | Number of components in the data structure definition.                                                        |
+  | Components                            | List of components with component property, property label, property type, column title and required columns. |
+  | Columns where suppress output is true | List of columns where the suppress output is true.                                                            |
+
+* `The data cube has the following code list information`:
+
+  | Field                | Description                                                                 |
+  | -------------------- | --------------------------------------------------------------------------- |
+  | Number of Code Lists | Number of code lists in the data cube.                                      |
+  | Code Lists           | List of codelists with code list uri, code list label, and columns used in. |
+  
+* `The data cube has the following dataset information`:
+
+  | Field                  | Description                                         |
+  | ---------------------- | --------------------------------------------------- |
+  | Number of Observations | Number of observations (i.e. rows) in the data set. |
+  | Number of Duplicates   | Number of duplicated observations in the data set.  |
+  | First 10 Observations  | Sample consists of first 10 obervations.            |
+  | Last 10 Observations   | Sample consists of last 10 obervations.             |
+
+* `The data cube has the following value counts`- Data set value counts broken-down by measure and unit (of measure).
+
+### Code list
+
+`csvcubed inspect --log-level info ./codelist.csv-metadata.json`
```

Comment:
Again, let's get rid of the `--log-level` param (and the associated output) to keep it simpler.